packages/fxa-content-server/tests/server/amplitude.js

```diff
@@ -2175,5 +2175,395 @@ registerSuite('amplitude', {
       assert.equal(arg.event_type, 'fxa_connect_device - submit');
       assert.equal(arg.event_properties.connect_device_flow, 'pairing');
     },
+
+    'screen.get-started': () => {
+      amplitude(
+        {
+          time: '1585321743',
+          type: 'screen.get-started',
+        },
+        {
+          connection: {},
+          headers: {
+            'x-forwarded-for': '63.245.221.32',
+          },
+        },
+        {
+          flowBeginTime: '1585261624219',
+          flowId:
+            '11750082326622a61b155a58a54442dd3702fa899b18d62868562ef9a3bc8484',
+          uid: '44794bdf0be84d4e8c7a8026b8580fa3',
+        }
+      );
+
+      assert.equal(logger.info.callCount, 1);
+      const arg = logger.info.args[0][1];
+      assert.equal(arg.event_type, 'fxa_qr_connect_device - get_started_view');
```

Comment:
If `logger.info` here is a sinon spy/stub, could alternatively use `sinon.assert.calledOnceWithExactly(logger.info, [args])`.

Reply:
Gonna hold off on this change for now